api_db.php

```diff
@@ -94,16 +94,33 @@ function SQLite3_connect($trytoreconnect)
 	{
 		$limit = "WHERE timestamp <= ".$_GET["until"];
 	}
-	$results = $db->query('SELECT client,count(client) FROM queries '.$limit.' GROUP by client order by count(client) desc limit 10');
+	$results = $db->query('SELECT client,count(client) FROM queries '.$limit.' GROUP by client order by count(client) desc limit 20');
 
 	$clients = array();
 
 	if(!is_bool($results))
 		while ($row = $results->fetchArray())
 		{
-			$clients[$row[0]] = intval($row[1]);
-			// var_dump($row);
+			// Convert client to lower case
+			$c = strtolower($row[0]);
+			if(array_key_exists($c, $clients))
+			{
+				// Entry already exists, add to it (might appear multiple times due to mixed capitalization in the database)
+				$clients[$c] += intval($row[1]);
+			}
+			else
+			{
+				// Entry does not yet exist
+				$clients[$c] = intval($row[1]);
+			}
 		}
+
+	// Sort by number of hits
+	arsort($domains);
+
+	// Extract only the first ten entries
+	$domains = array_slice($domains, 0, 10);
```

Comment:
Copy-paste error. Should be operating on `$clients`

Reply:
No, it shouldn't?

Reply:
Probably was a force-push